tests/Maker/MakeMigrationTest.php

```diff
@@ -76,7 +76,15 @@ public function getTestDetails()
             ->assert(function (string $output, string $directory) {
                 $this->assertStringContainsString('You have 1 available migrations to execute', $output);
                 $this->assertStringContainsString('Success', $output);
-                $this->assertCount(14, explode("\n", $output), 'Asserting that very specific output is shown - some should be hidden');
+
+                $output = explode("\n", $output);
+
+                $expectedCount = str_contains($output[0], 'php') ? 15 : 14;
```

Comment:
Would it be better to filter out the new line? Especially if the shebang is printing to the console, we don't really want that.